public/_posts/_articles/marco-worms/self-hosting/es.md

```diff
@@ -0,0 +1,241 @@
+---
+title:  "Servicios Web3 de alojamiento propio"
+image:
+  src: ./image1.jpg
+  width: 1400
+  height: 702
+date: '2022-03-31'
+author: Marco_Worms
+translator: RestlessMik3
+---
+
+![](./image1.jpg?w=1400&h=702)
+
+Web3 ha traído una nueva forma de comunicarse con las aplicaciones: las aplicaciones ya no están alojadas en proveedores de servidores centralizados, sino en cadenas de bloques, que son sistemas descentralizados. Las aplicaciones construidas sobre bases descentralizadas son resistentes al tiempo de inactividad y la censura, pero hay un problema: servir la interfaz de la aplicación a los usuarios que usan el navegador web a menudo depende de entidades centralizadas.
+
+En este artículo, aprenderemos a autohospedar las interfaces de las aplicaciones para los servicios cripto principales y de Yearn, por lo que si los sitios web oficiales están caídos por algún motivo, ¡puedes servir el sitio web por ti mismo ya que la cadena de bloques nunca está caída!
+
+## Back-end, Front-end, Web3!
+
+Una introducción rápida a estos conceptos.:
+
+- **Front-end** es el nombre que se le da a la parte de la aplicación que ve el usuario, como un sitio web o una aplicación móvil.
+- **Back-end** es el nombre que se le da a la parte de la aplicación que el usuario no ve. Muchas acciones de front-end dependen del back-end para procesar.
```

Comment:
AS-IS: - **Back-end** es el nombre que se le da a la parte de la aplicación que el usuario no ve. Muchas acciones de front-end dependen del back-end para procesar.

TO-BE: - **Back-end** es el nombre que se le da a la parte de la aplicación que el usuario no ve. Muchas acciones de front-end dependen del back-end para ser procesadas.

Reply:
Done